.github/workflows/typos.yml

```
@@ -0,0 +1,15 @@
+name: Typos Workflow
+on:
+  push:
+  pull_request:
+jobs:
+  typos-check:
+    name: Spell Check with Typos
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout Actions Repository
+        uses: actions/checkout@v3
```

```suggestion
        uses: actions/checkout@v4
```
We can start to use v4 now.